lib/ASRouter.jsm

```diff
@@ -690,18 +690,34 @@ class _ASRouter {
   }
 
   async _getBundledMessages(originalMessage, target, trigger, force = false) {
```

Comment:
Unrelated to your changes, but `target` isn't used inside this function.

Reply:
ah weird